app/models/inning.rb

```diff
@@ -1,5 +1,6 @@
 class Inning < ApplicationRecord
   validates_presence_of :name
+  validates_presence_of :start_date
```

Comment:
This validation makes sense. Let's put it into a common separated list with :name

Reply:
That makes total sense, I always forget you can in line validate multiple fields.

Reply:
Fixed